module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,9 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> words){
+    words.sort();
```

Comment:
Will it be sorted lexicographically?

Reply:
This will not works. Try to add more tests :) This is special tricky exercise :)